spec/drag-and-drop.spec.js

```diff
@@ -19,14 +19,6 @@ describe('Drag and Drop TestCase', function () {
             expect(editor.elements[0].className).toContain('medium-editor-dragover');
         });
 
-        it('should add medium-editor-dragover class even when data is invalid', function () {
```

Comment:
There are 2 separate concepts with the drag-and-drop.

1) Without the extension, drag and drop will cause the browser to open the file by navigating to the file or opening the file in a separate tab.  When the extension is instantiated, it listens for the drop event over the editor and prevents the default action, which would be having the browser open the file.
2) The `allowedType` specifies that for a certain file type, the code should have some special handling for inserting the file into the editor.  Currently, we only have code that handles images, so `'image'` is the only meaningful value to pass into this array.

So, `allowedTypes` should not affect whether the extension exists or not, since it is intended for a different purpose (the parameter is a bit confusing and probably named incorrectly, that's my fault and I hope to fix it for 6.0.0).